### DISCUSSION TITLE
sql-formatter setup

### DISCUSSION THREAD

**Discussion by deng232** [LazyVim/LazyVim#1742] [Q&A] [answered]
I have sql-formatter installed through ```Mason```, but showing no formatter found in the use. 
I see there is an old post using ```null-ls```, but I'm wonder what to do with the current version using ```conform```

I'm using the ```https://github.com/LazyVim/starter```
and I have tried to add ```conform.lua``` file in the ```plugins``` folder but still has no formatter found for sql.
```
return {
  "stevearc/conform.nvim",
  formatters_by_ft = {
    sql = { "sql-formatter" },
  },
}
```

**Comment by tby321** [comment]
sql_formatter

**Comment by dadav** [comment, marked as answer]
You must write it like this

```lua
return {
{
  'stevearc/conform.nvim',
  opts = {
    formatters_by_ft = {
      sql = {"sql_formatter"}
    },
  },
}
}
```

**Reply by arieroos** [reply]
I completely missed how your answer actually changed the config in question, for longer than I care to admit.

So just to spell it out more clearly for future readers like me, who might not be paying close attention: The difference is that you need `sql[UNDERSCORE]formatter`, not `SQL[DASH]formatter`.

**Reply by nxtkofi** [reply]
no, extra brackets.
